page.php

```diff
@@ -76,6 +76,7 @@
 
 // P4 Campaign/dataLayer fields.
 $context['cf_campaign_name'] = $page_meta_data['p4_campaign_name'][0] ?? '';
```

Comment:
Probably we want to deduplicate these 3 files here, maybe I'll add that to this PR too.